packages/components/src/styles/components/form/layout.scss

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM LAYOUT COMPONENTS
+//
+
+// FORM
+
+// Default max-width for form sections & similar components
+$hds-form-section-max-width: 768px;
+
+.hds-form {
+  --hds-form-section-max-width: #{$hds-form-section-max-width};
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  // Set a max-width for the ButtonSet when it's within an Hds::Form
+  // Q: Should this be done (max-width on ButtonSet) and if so should it be done here or in the ButtonSet component?
```

Comment:
[thought] what if we bypass completely the problem, and have the consumers wrap the button set in a `Form::Section` if they really need to set a max-width to it (which I don't think will ever be the case)? plus, if we add the `hasMaxWidth` also at form level, this would become an ever more edge case